app/views/layouts/application.html.erb

```diff
@@ -3,6 +3,8 @@
 <head>
   <title>Bubble Tea Shop Reviews</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Fredoka+One">
```

Comment:
this should go in our CSS instead so that all styling lives close. Like I mentioned above, I recommend using `.scss` rather than `.css` - one thing it allows us to do is import fonts:
```
@import url('https://fonts.googleapis.com/css2?family=Fredoka+One');
```